tpm.go

```diff
@@ -178,7 +177,7 @@ func DecodeEK(pemBytes []byte) (*attest.EK, error) {
 
 	switch block.Type {
 	case "CERTIFICATE":
-		cert, err := certx509.ParseCertificate(block.Bytes)
+		cert, err := x509.ParseCertificate(block.Bytes)
```

Comment:
🤔 I was wondering if dropping the certificate-transparency lib here may bring any difference... but since _ParseCertificate_ is just parsing the cert, I guess it makes no difference at all 👍🏼

Reply:
Yep, I guess it was changed upstream since it was not building otherwise.. :+1: